dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -1091,5 +1091,105 @@ public void VerifyHierarchicalNavigationTopModeMenuItemsSourceDoesNotCrash()
                 Content.UpdateLayout();
             });
         }
+
+        [TestMethod]
+        public void VerifyNavigationViewItemToolTipCreation()
+        {
+            if (!PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone5))
+            {
+                Log.Warning("On RS4 and earlier the test needs to be modified slightly.");
```

Comment:
This should only be a placeholder for now. The tooltip implementation changed with RS5+ and I can only test that behavior on my local machine for now. I do not have a VM up and running RS4 and earlier yet. Since I cannot locally test that scenario, I haven't written a test case for it yet, though the implementation used in RS4 and below [still exists](https://github.com/microsoft/microsoft-ui-xaml/blob/f5034668462987302f1e1c58d1e055e45e40c658/dev/NavigationView/NavigationViewItem.cpp#L367) and I could write a test based on the source code. Testing that would have to rely on the test pipeline then to spot any errors made when writing that test.

Should I go ahead and create a test based on the available implementation?